OJP/OJP_Booking.xsd

```diff
@@ -0,0 +1,596 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified">
+	<xs:annotation>
+		<xs:documentation>OJP/OJP_Booking.xsd - Request and response definitions for processing bookings.
+            It was decided that only Availability is modeled in OJP.</xs:documentation>
```

Comment:
What does "It was decided that only Availability is modeled in OJP" mean?
Should the booking requests be removed again?
Anyway this is definitely the wrong place for a comment like this, as it will be rendered in the documentation header of the OJP_Booking.xsd file:

https://htmlpreview.github.io/?https://github.com/VDVde/OJP/blob/7da0bc881e4c6fdcc7606116ebfe45d84c3fe002/docs/generated/OJP.html

Reply:
This is indeed very misleading then. Why is there OJP_Booking.xsd then? Why is there a `BookingRequestStructure` if it is not used?

Reply:
The file name and structures align with TRIAS, in my view it makes sense to keep it like that.

The question about the necessity of *BookingRequestStructure* is pertinent. There are more structures like that (e.g., *BookingCanellationRequestStructure*) that are not needed as long as no "real" booking is to be supported. 

I don't know how exactly they arrived here. I remember having added code for *AvailabilityRequest* last year, working on some remote separate branch. Manoeuvring those changes to the right place might have brought more code in than wanted? Shall we keep as much as possibly useful, or, on the contrary, remove everything unnecessary?

Reply:
I am in favor of removing the unnecessary requests.

Reply:
As discussed in the meeting - removed the booking stuff that is currently not necessary (commits https://github.com/VDVde/OJP/pull/262/commits/6d2429356f4fadf4344aff69a6c826d372ff2aca and https://github.com/VDVde/OJP/pull/262/commits/c8e1359aef47602763e47d174d5acdbe3feac86b).